### PR TITLE
Add cookie handling and user levels

### LINKED ISSUE #33: Hide disallowed actions on frontend
Now that we have FB auth, we need to make sure that we show the relevant elements only.

### PR DESCRIPTION
**Pull request by thiderman**
Closes #33

### CODE REVIEW

File: js/src/components/Match.vue [hunk 1]
```diff
@@ -39,7 +41,9 @@ export default {
   data () {
     return {
       match: new Match(),
-      tournament: new Tournament()
+      tournament: new Tournament(),
+      user: this.$root.user,
+      levels: levels,
```
Comment (thiderman) [inline js/src/components/Match.vue]: @frontierpsycho: Is this an acceptable way of adding the `levels` constants into the apps? I wanted to find a way where we didn't have to do that everywhere but they would be accessible anywhere, but I was not able to!

Comment (frontierpsycho) [inline js/src/components/Match.vue]: Reading the vue docs, I'd say yes.

Comment (thiderman) [inline js/src/components/Match.vue]: Aight, then I made a semi-correct choice anyway!
Good thing it's not much to copy-paste!

Comment (frontierpsycho) [inline js/src/components/Match.vue]: To be honest, it feels a bit dubious, but I couldn't find a better way either :-/

But it works fine and isn't that hard!

**Pull request merged**